.env.sample

```diff
@@ -1 +1,7 @@
 FIREBASE_API_KEY=[put your key here]
+FIREBASE_AUTH_DOMAIN=[put your key here]
```

Comment:
I would make this the real `.env` file instead of a sample. Perhaps `.env`

We should then add it gitignore so people aren't commiting their `.env` file updates. 

If people know what they're doing and want to make changes to the real `.env` you can `--force` add.

Reply:
I think it's already `.gitignored` IIRC, no? Is there any benefit to making it a real file?

Reply:
I'm neutral on this. I set it up this way and it's documented in the README. Whatever decision is reached, please update the README accordingly

Reply:
this sample file isn't but a root `.env` is now. 

I'm thinking do a real one because we'll have more values than just firebase keys in the `.env` file. I'm already working on a PR that will need it.

Reply:
I'm personally more a fan of the way @ann-kilzer has set it up, as if we want to add more keys in the future, it will be much easier to keep track of them if we have `.env.sample` as a running reference. If we commit an `.env` file, we run into the choice of either a) ignoring all updates to necessary `.env` keys and not having a good reference or b) clobbering everyone's existing files whenever we decide to do an update.

Reply:
hmmm.....that's a fair point. I guess in either case, if we add keys with this approach it'll be tough. we'd have to either make people copy new changes in `.env.sample` or deal with merge issues. 🤔 

typically this is solved by having two `.env` files. `.env.prod/default` and `.env.local`. 

How about this....we stick to your method for generating/maintaining the `.env.local` file and we made a new `.env.default/prod/whatever` later for default values? (I need to add some this weekend)

Reply:
If we were going to setup different runtime environments (ie, `dev`/`integ`/`prod`, etc) that would make sense, but I don't know if it might be too much indirection for what we're doing here. I think that's a major change to our build process that aught to be tracked in a separate issue / PR to preserve discussion, rather than attached to this one.

>  we'd have to either make people copy new changes in .env.sample

This is the way we're currently set up, which seems ok IMO for now. Granted, we opted for this because we needed the secrets outta there, like, _now_, but I'm not sure it's such a bad pattern - it allows us to both track what _aught_ to be in the `.env` file while at the same time allowing for local config.